Static/DataSource.swift

```diff
@@ -311,6 +311,7 @@ extension UITableView.Style {
         switch self {
         case .plain: return 0
         case .grouped: return UITableView.automaticDimension
+        @unknown default: return UITableView.automaticDimension
```

Comment:
Nice!

Reply:
I took a moment to think which value should be returned for default, `0` vs `UITableView.automaticDimension`. I have no idea about future of `UITableView.Style`, so I hope `UITableView.automaticDimension` would handle the default case nicely 😛